packages/xsnap/src/avaHandler.js

```diff
@@ -1,4 +1,4 @@
-/* global globalThis __dirname __filename */
+/* global globalThis */
```

Comment:
It doesn't seem fair to remove `__filename` from our "HERE BE DRAGONS!" ambient authority declaration without adding `import` (for `import.meta.url`) while we're at it. Hm. I suppose I'll open a separate issue for that.

Reply:
Unfortunately, `import` is the kind of ambient authority that flows from a keyword and not a global. It can’t even be overshadowed, but SES tames it into oblivion regardless.

I won’t be able to add `import` to the globals in any case because it would be an unreferenced global as far as the linter is concerned.